### DISCUSSION TITLE
Publishing win-ARM64 fails with 5.5

### DISCUSSION THREAD

**Discussion by TopperDEL** [unoplatform/uno#18681] [General]
I have a build pipeline with github actions that automatically builds for Win-ARM64 (and others). After updating to Uno 5.5 (from 5.4) it fails with this error:

`C:\Users\runneradmin\.nuget\packages\uno.sdk.extras\5.5.10\buildTransitive\Uno.Sdk.Extras.Publish.Windows.targets(19,3): error : Cross publishing self-contained WPF applications is not supported. `

The corresponding command in my YAML is:

```
- name: Build MAUI Windows arm64
   run: dotnet publish MyApp.csproj -c Release -f net8.0-windows10.0.19041 -p:Platform=arm64 -r win-arm64 --self-contained /p:GenerateAppxPackageOnBuild=true
```

Any ideas?

**Comment by TopperDEL** [comment]
Hm, it seems removing the "--self-contained" fixes it. At least it could be built again.

**Reply by jeromelaban** [reply]
Thanks for the report! Let's track in https://github.com/unoplatform/uno/issues/18682.
